core/src/main/java/org/arquillian/smart/testing/hub/storage/local/LocalStorage.java

```diff
@@ -7,6 +7,8 @@
  */
 public class LocalStorage {
 
+    public static String JUNIT_5_PLATFORM_VERSION = "junit5PlatformVersion";
```

Comment:
Minor thing - I'm not sure this should belong here - it's not related to storage at all. Maybe `LoaderVersionExtractor` would be a better place?

Reply:
The reason why we put it here is that we need to have it available on both sides - maven extension and surefire
We had the same discussion - where we should put it and we decided for this place

Reply:
Don't we have `LoaderVersionExtractor` available for both side though?